beesdoo_print_label/__manifest__.py

```diff
@@ -18,4 +18,6 @@
         "data/product_print_category.xml",
     ],
     "installable": True,
+    "application": True,
```

Comment:
An application covers a new functional scope and gets its entry point in the menu. This is not an application but only an addon.